mentat/conversation.py

```diff
@@ -246,29 +271,10 @@ async def get_model_response(self) -> ParsedLLMResponse:
         session_context = SESSION_CONTEXT.get()
         stream = session_context.stream
         config = session_context.config
-        code_context = session_context.code_context
 
-        messages_snapshot = self.get_messages()
-
-        # Get current code message
-        prompt = messages_snapshot[-1].get("content")
-        if isinstance(prompt, list):
-            text_prompts = [
-                p.get("text", "") for p in prompt if p.get("type") == "text"
-            ]
-            prompt = " ".join(text_prompts)
-        code_message = await code_context.get_code_message(
-            prompt_tokens(messages_snapshot, config.model),
-            prompt=(
-                prompt  # Prompt can be image as well as text
-                if isinstance(prompt, str)
-                else ""
-            ),
-        )
-        messages_snapshot.insert(
-            0 if config.no_parser_prompt else 1,
-            ChatCompletionSystemMessageParam(role="system", content=code_message),
-        )
+        messages_snapshot = await self.get_messages(include_code_message=True)
+        tokens_used = prompt_tokens(messages_snapshot, config.model)
```

Comment:
```suggestion
        tokens_used = await self.count_tokens(include_code_message=True)
```
Can this new method be reused here?

Reply:
It could be used but we still need to compute `messages_snapshot` to send to the llm anyway. So I think it is better as is.